src/crate/client/sqlalchemy/types.py

```diff
@@ -108,6 +108,9 @@ class Comparator(sqltypes.TypeEngine.Comparator):
         def __getitem__(self, key):
             return self._binary_operate(self.expr, operators.getitem, key)
 
+    def get_col_spec(self):
+        return 'object'
```

Comment:
a little picky: `OBJECT` because all other keywords are uppercase too.
(changes the output of `create_table_test.py:53`